internal/controllers/user_integrations_controller_test.go

```diff
@@ -101,12 +101,15 @@ func (s *UserIntegrationsControllerTestSuite) SetupSuite() {
 		s.autoPiIngest, s.deviceDefinitionRegistrar, s.autoPiTaskService, nil, nil, s.drivlyTaskSvc, nil, s.redisClient, nil, nil)
 
 	app := test.SetupAppFiber(*logger)
+
 	app.Post("/user/devices/:userDeviceID/integrations/:integrationID", test.AuthInjectorTestHandler(testUserID), c.RegisterDeviceIntegration)
-	app.Post("/user2/devices/:userDeviceID/integrations/:integrationID", test.AuthInjectorTestHandler(testUser2), c.RegisterDeviceIntegration)
-	app.Get("/integrations", c.GetIntegrations)
 	app.Post("/user/devices/:userDeviceID/autopi/command", test.AuthInjectorTestHandler(testUserID), c.SendAutoPiCommand)
 	app.Get("/user/devices/:userDeviceID/autopi/command/:jobID", test.AuthInjectorTestHandler(testUserID), c.GetAutoPiCommandStatus)
+
```

Comment:
Why'd we move these?

Reply:
I initially moved these because I was trying to incorporate the middleware tests into these and I wanted to visually distinguish what was being grouped (and tested with middleware) the ones I moved all had `/user/devices/:userDeviceID` and were injecting `testUserID` with `AuthInjectorTestHandler`